view/frontend/web/js/view/payment/abstract-method.js

```diff
@@ -136,8 +114,10 @@ define([
         },
 
         initPayment: function () {
+            if(!this.intentConfiguration().id){
```

Comment:
What if this.intentConfiguration is null? Please check for parent types.

Reply:
Will proceed to fix this